JumpscaleCore/tools/threebot/ThreebotMe.py

```diff
@@ -32,6 +32,10 @@ def sign(self, data):
         raise
         # TODO: implement
 
+    def sign_bytes(self, data):
```

Comment:
since ultimately you need to send the signature hex encoded, why not just have this method return it hex encoded directly?

Reply:
Because the function itself is expected to return just the signed bytes directly if used in another scenario involving 3bot other than the reservation.